sbnana/CAFAna/Analysis/Fit.h

```diff
@@ -238,6 +238,18 @@ namespace ana
                std::map<const IFitVar*, TGraph*>& profVarsMap = empty_vars_map,
                std::map<const ISyst*, TGraph*>& systsMap = empty_syst_map);
 
+  TH1* Profile(const IExperiment* expt,
+	       osc::IOscCalcAdjustable* calc,
```

Comment:
something has gone wrong with the indentation of the arguments here? At least on github's view. I suspect there are tabs.